### PR TITLE
Added additional support for postback related calls in RegistrationServices

### PR DESCRIPTION
**Pull request by musicm122**
## Added the following methods to Registration service:
- GetPostbackInfo
- UpdatePostbackInfo
- DeletePostbackInfo
- TestRegistrationPostUrl
Added 2 classes for parsing xml
- PostbackInfo
- Rsp

## Misc Fixes:
- Fixed spelling errors
- Fixed parameter issues in GetPostbackInfo call

### CODE REVIEW

**Comment by musicm122** [comment]
Sorry about the repeat pulls. Its been one of those days...

File: RegistrationService.cs [hunk 1]
```diff
@@ -521,5 +568,67 @@ public LaunchInfo GetLaunchInfo(string launchId)
...
+	    /// <param name="urlname">You can optionally specify a login name to be used for credentials when posting to the URL specified in postbackurl</param>
+	    /// <param name="urlpass">If credentials for the postbackurl are provided, this must be included, it is the password to be used in authorizing the postback of data to the URL specified by postbackurl</param>
+	    /// <param name="resultsFormat">This parameter allows you to specify a level of detail in the information that is posted back while the course is being taken. It may be one of three values: "course" (course summary), "activity" (activity summary, or "full" (full detail), and is set to "course" by default. The information will be posted as xml, and the format of that xml is specified below under the method "getRegistrationResult"</param>
+	    /// <returns>Rsp containing result and status code</returns>
+	    public Rsp TestRegistrationPostUrl(string postbackUrl, RegistrationResultsAuthType authType, string urlname, string urlpass, RegistrationResultsFormat resultsFormat)
```
Comment (nathaninlatin) [inline RegistrationService.cs]: authType is in the parameter list, so it should either be deleted or added as a parameter in the body of this function.

**Comment by nathaninlatin** [comment]
Hi, thanks for this PR. Looks good. I had just the one comment.

**Comment by musicm122** [comment]
K updated.

**Comment by nathaninlatin** [comment]
Hi, so Github's display of my comment was kind of funny. I was commenting on the second TestRegistrationPostUrl function, which already had authType in the parameter list, but which didn't use it. I didn't mean for you to add authType to the other TestRegistrationPostUrl function. It is now on line 611 of RegistrationService.cs…authType is passed in as a parameter to the function but not added as a parameter to the API call.

**Comment by musicm122** [comment]
K fixed.

**Comment by nathaninlatin** [comment]
Thanks! :+1:

**Pull request merged**